### ISSUE TITLE
introduce Table View

### ISSUE THREAD

**Issue by eMerzh** [sqlectron/sqlectron-gui#187]
I'm a bit confuse about the absence of "table view" in sqlectron..
probably due to his age :) 
here is my "mock-up" take on the table view:

the idea is to have at least 2 tabs, one with datas the other with structures and other infos....
later we can add relations, indexes, or other info in tabs...

what do you think about this...
these are only the first thoughts

<img width="676" alt="capture d ecran 2016-06-05 a 22 52 17" src="https://cloud.githubusercontent.com/assets/177003/15808147/72154316-2b70-11e6-85b5-2b630885ab7c.png">
(of course this is the version with both tabs clicked)

**Comment by maxcnunes** [comment]
This feature would aggregate a lot to the app. Although it is something usually I don't really use in a SQL client. Because of that I'm not interesting im coding it my self. But I would gladly accept a PR 😄 

As I said I have not used so much this kind of feature in other SLQ clients. So I have some questions:

1. Does the "Info" tab show the information about the columns such as: name, type, if is NULLABLE and etc? I'm asking that because that is what I have seen in other SQL clients. But looking your prototype looks like is listing the table data (all the rows).
1. Could we have 3 tabs instead of only 2? I guess splitting the 2 boxes of the first tab in different tabs would give a better look. Although if the second box in the first tab is always small than makes sense keep them together.

**Comment by eMerzh** [comment]
yeah here is an updated version ...

i understand that it wasn't really a priority ... but i wanted to know what the team think about it ...

For the first question, i was thinking about the sql description of the create table with nullable, with default, ... 

And for the 3rd tab, yeah i think it's a good idea... not sure yet wat we can / can't put in it ... but 
i was wondering if we couldn't also add a "Query" tab which will give the current Query + data view...

<img width="669" alt="capture d ecran 2016-06-06 a 09 14 52" src="https://cloud.githubusercontent.com/assets/177003/15814541/8ffc95b2-2bc7-11e6-8c3b-f1677941a832.png">

**Comment by maxcnunes** [comment]
Let me se if I have understood:

* Tab 1 - Structure: Show the SQL script of the table structure.
* Tab 2 - Content: Show the table data. With a `WHERE` filter?
* Tab 3 - Info: Show information about the table. Such as the size, number of rows, and etc.

If the Tab 2 really shows the table data I think that is not gonna work for large tables. Otherwise we would need paginate the result to not causa a memory leak. But I prefer keep it simple and do not have this tab. Because is possible to use "select rows (with limit)" to achieve almost the same thing.

**Comment by eMerzh** [comment]
@maxcnunes 
my idea wat to give content tab when you double click on the table in the sidebar...
un filtered but limited to 1000 records or so...

then you have the choice to filter using the where field (sort of search)  or to create your own query as now... with select rows (with limit).

but i understand that it might not be the point of the tool

**Comment by BornaP** [comment]
If we are ever going to have DB statistics (like allocation on disk, pages and stuff) I think better feature would be to make it visualized. Pure data in text form can be retrieved through SQL anyways (and is available in every SQL editor). On the other hand, for example, showing disk allocation per table as tree map graph would make much more sense for the end user :)

But as someone said earlier... this is currently low priority I guess.

**Comment by eMerzh** [comment]
@BornaP yeah why not tree map ...  i was also thinking about table relationships with other table (using the PR about relationships)

**Comment by maxcnunes** [comment]
Well, that is a lot of stuff 😄  Maybe starting with a single tab (the one that is more useful for you) I guess would be better.

**Comment by BornaP** [comment]
@eMerzh As Max said, one thing at the time. Start with a single feature you find most useful :)

**Comment by eMerzh** [comment]
yeah i know... this was more about the "big plan" ... seeing if we somewhat agree on the feature..
i'm interested in developping it, but not sure i have the time :)
i'll start small :p

**Comment by rathboma** [comment]
I love this. I used to use Sequel Pro a lot when I was a heavy Mac+MySQL user, and their table-information and data display panes are very useful.
